src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+form {
+  margin: 30px 0;
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+}
+
+.big {
+  width: 100%;
```

Comment:
You could use mix of classes, for example `class="input input-big`
And use common styles for all inputs(like width, background, box-sizing, border-radius etc.) in `.input` class and other specific classes in `input-big` and `.input-small`
Like you've done with images